generic/secrets/security/detected-aws-account-id.yaml

```diff
@@ -1,8 +1,23 @@
 rules:
   - id: detected-aws-account-id
-    pattern-regex: |-
-      ("|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?("|')?\s*(:|=>|=)\s*("|')?[0-9]{12}("|')?
-    languages: [regex]
+    patterns:
+    - pattern-either:
+      - pattern: |
+          $ACCOUNT_ID = $SECRET
```

Comment:
Same as the other rule - this could be in a URL somewhere. Maybe we don't care about it sometimes, but I'm curious how the old and this new rule look when tried at scale.

Reply:
from the 400 repos we tested on, there were no URL results. I'll add in string-checking, though 👍